docs/known_issues.rst

```diff
@@ -226,6 +226,18 @@ means that an upstream fix in ``numpy` is required in order for
 To convert a dimensionless `~astropy.units.Quantity` to an integer, it is
 therefore recommended to use ``int(...)``.
 
+Inconsistent behavior when converting complex numbers to floats
+---------------------------------------------------------------
+
+Attempting to use `float` or NumPy's `~numpy.float` on a standard
+complex number (e.g., `5 + 6j`) results in a `TypeError`.  In
```

Comment:
use double backticks around non API linkable things, e.g. the ``5 + 6j``

Reply:
Thanks!  I'll tick this one off my list.